ludwig/features/base_feature.py

```diff
@@ -558,4 +558,8 @@ def type():
         def unskip(self) -> InputFeature:
             return feature
 
+        @property
+        def encoder_obj(self) -> torch.nn.Module:
```

Comment:
I would like to, but it's a bit tricky, because for most modules it's an attribute, not a propetty (we use a property here to avoid setting it as an attribibute, which would put it in this module's registry in PyTorch, which would cause it to be treated as paramters sent ot the otpimizer, which we don't want). Ideally this would be a property, but since it's not on other modules, it means that we would need to rename the attribute to like `_encoder_obj`, which would complicate checkpoint laoding for inference (because PyTorch will expect an attribute named `encoder_obj`). Sadly one of the downsides to PyTorch.